src/user/user.helpers.ts

```diff
@@ -0,0 +1,95 @@
+import HttpException from '../exceptions/HttpException';
```

Comment:
This `user.helpers.ts` looks like a typical "service" file.
Therefore, I'd suggest renaming all similar file into `*.service.ts`.
Usually, in TS context these files contain a named `*Service` class with its main operations stored as methods.